PHP 8.1 support

boesing opened the pull request:
|    Q          |   A
|-------------- | ------
| New Feature   | yes

### Description

Initial PHP 8.1 support.

boesing committed:
qa: add PHP 8.1 to `composer.json`

Signed-off-by: Maximilian Bösing <2189546+boesing@users.noreply.github.com>

boesing committed:
qa: add/modify `.laminas-ci.json` to ignore PHP platform requirements for PHP 8.1

Signed-off-by: Maximilian Bösing <2189546+boesing@users.noreply.github.com>

boesing committed:
qa: remove `laminas/laminas-zendframework-bridge` and conflict with zendframework pendant

Signed-off-by: Maximilian Bösing <2189546+boesing@users.noreply.github.com>

Ocramius commented on composer.json:
```diff
@@ -18,13 +18,11 @@
         "laminas/laminas-math": "^2.7 || ^3.0",
-        "laminas/laminas-stdlib": "^3.3",
-        "laminas/laminas-zendframework-bridge": "^1.1"
+        "laminas/laminas-stdlib": "^3.3"
```
This should be bumped to `^3.6`

boesing committed:
qa: bump `laminas/laminas-stdlib`

Signed-off-by: Maximilian Bösing <2189546+boesing@users.noreply.github.com>

boesing commented:
Waiting for https://github.com/laminas/laminas-session/pull/45 which is waiting for https://github.com/laminas/laminas-db/pull/204

This package would be ready for PHP 8.1 if these PRs were merged. So no further deprecations within **THIS** package.

weierophinney committed:
qa: bump laminas-session dep to 2.12

First version supporting PHP 8.1.

Signed-off-by: Matthew Weier O'Phinney <matthew@weierophinney.net>

weierophinney committed:
fix: ensure we have integers for x/y coordinates

Operations were creating floats, which were then being passed where integers were needed.

Signed-off-by: Matthew Weier O'Phinney <matthew@weierophinney.net>

weierophinney committed:
qa: resolve psalm errors

- Remove unused variables.
- Ensure options keys are strings before passing them to internal methods.

Signed-off-by: Matthew Weier O'Phinney <matthew@weierophinney.net>
